e2e_playwright/conftest.py

```diff
@@ -381,11 +381,12 @@ def iframed_app(page: Page, app_port: int) -> IframedPage:
 worker-src blob:;
 form-action 'none';
 frame-ancestors {fake_iframe_server_origin};
-frame-src data: {app_url}/_stcore/component/;
+frame-src data: {app_url}/_stcore/component/ {app_url}/component/;
```

Comment:
This is an interesting aspect. The CSP in SiS only has `/_stcore/component/` but Streamlit OSS is actually requesting from `/component/`. Not sure why? Might need more follow up exploration.